relayer/relays/beacon/message/ethereum-listener.go

```diff
@@ -20,8 +21,13 @@ type ParachainPayload struct {
 	Messages []*chain.EthereumOutboundMessage
 }
 
+type EventContainer struct {
+	Event *etypes.Log
+	Nonce uint64
+}
```

Comment:
This is just a container struct to easily return the EthereumOutboundMessage nonce too.